index.html

```diff
@@ -25,12 +27,185 @@ <h2 class="padding">I’m Youssef Glad to see you!</h2>
+              <span class="card-title">Tonic</span>
+              <div class="card-info">
+                <span class="info-active">CANOPY</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>Back End Dev</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>2015</span>
+              </div>
+            </div>
+            <p class="card-descrip card-item-mg">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="card-tags card-item-mg">
+              <span class="card-tag">html</span>
+              <span class="card-tag">css</span>
+              <span class="card-tag">javascript</span>
+            </div>
+            <div class="card-action">
+              <button class="card-btn">See Project</button>
+            </div>
+          </div>
+        </div>
+        <!-- Fourth Card -->
+        <div class="card">
+          <img class="card-img" src="./assets/Snapshoot4.png" alt="" />
+          <div class="card-details">
+            <div class="card-item-mg">
+              <span class="card-title">Multi-Post Stories</span>
+              <div class="card-info">
+                <span class="info-active">CANOPY</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>Back End Dev</span>
+                <img src="./assets/Counter.svg" alt="" />
+                <span>2015</span>
+              </div>
+            </div>
+            <p class="card-descrip card-item-mg">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="card-tags card-item-mg">
+              <span class="card-tag">html</span>
+              <span class="card-tag">css</span>
+              <span class="card-tag">javascript</span>
+            </div>
+            <div class="card-action">
+              <button class="card-btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- About Me Section -->
+    <section class="about-me">
+      <h2 class="padding">
+        About <br />
+        Myself
+      </h2>
+      <p class="padding">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a class="padding" href="#">LET’S CONNECT</a>
+      <div class="social">
+        <img src="./assets/126.svg" alt="github" />
+        <img class="sc-item" src="./assets/125.svg" alt="linkedin" />
+        <img class="sc-item" src="./assets/123.svg" alt="hand" />
+        <img class="sc-item" src="./assets/127.svg" alt="twitter" />
+        <img class="sc-item" src="./assets/124.svg" alt="M" />
+      </div>
```

Comment:
- [ ] Please make sure that you use the `ul` and `li` tags to list the social media icons in your project. But you should use the `a` tag to wrap the `img` tag in your project. So that when you click on the social media icon it will take you to the social media page. :pray:

Reply:
fixed